velox/common/memory/SharedArbitrator.h

```diff
@@ -285,18 +285,36 @@ class SharedArbitrator : public memory::MemoryArbitrator {
     const std::chrono::steady_clock::time_point startTime_;
   };
 
+  // Object in whose scope global arbitration takes place. It ensures
```

Comment:
Can rephrase this a bit? sth like: The scoped object to cover the global arbitration execution. It setups ...